website/static/js/fangorn.js

```diff
@@ -1157,11 +1157,11 @@ function _fangornUploadMethod(item) {
     return configOption || 'PUT';
 }
 
-function gotoFileEvent (item) {
+function gotoFileEvent (item, toUrl) {
```

Comment:
I checked around and I think goToFileEvent is only used in this file, that's why I did it that way.